### PR TITLE
Convert controller bootstrap files to structured logs

### PR DESCRIPTION
**Pull request by scottilee**
#### What type of PR is this?
/kind cleanup
/sig instrumentation

#### What this PR does / why we need it:
Converts log statements to use structured logging.

#### Which issue(s) this PR fixes:
N/A

#### Special notes for your reviewer:
N/A

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```docs
- [KEP]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1602-structured-logging/kep.yaml
```

### CODE REVIEW

File: pkg/controller/bootstrap/bootstrapsigner.go [hunk 1]
```diff
@@ -200,7 +200,7 @@ func (e *Signer) signConfigMap() {
 	// First capture the config we are signing
 	content, ok := newCM.Data[bootstrapapi.KubeConfigKey]
 	if !ok {
-		klog.V(3).Infof("No %s key in %s/%s ConfigMap", bootstrapapi.KubeConfigKey, origCM.Namespace, origCM.Name)
+		klog.V(3).InfoS("No key in ConfigMap", "configKey", bootstrapapi.KubeConfigKey, "configMapNamespace", origCM.Namespace, "configMap", origCM.Name)
```
Comment (yangjunmyfm192085) [inline pkg/controller/bootstrap/bootstrapsigner.go:203]: Is klog.KObj or klog.KRef  feasible？

File: pkg/controller/bootstrap/bootstrapsigner.go [hunk 2]
```diff
@@ -297,7 +297,7 @@ func (e *Signer) getTokens() map[string]string {
 		if _, ok := ret[tokenID]; ok {
 			// This should never happen as we ensure a consistent secret name.
 			// But leave this in here just in case.
-			klog.V(1).Infof("Duplicate bootstrap tokens found for id %s, ignoring on in %s/%s", tokenID, secret.Namespace, secret.Name)
+			klog.V(1).InfoS("Duplicate bootstrap tokens found for id, ignoring in Secret", "tokenID", tokenID, "secretNamespace", secret.Namespace, "secret", secret.Name)
```
Comment (yangjunmyfm192085) [inline pkg/controller/bootstrap/bootstrapsigner.go:300]: Is klog.KObj or klog.KRef  feasible？

File: pkg/controller/bootstrap/tokencleaner.go [hunk 3]
```diff
@@ -191,7 +191,7 @@ func (tc *TokenCleaner) evalSecret(o interface{}) {
 	secret := o.(*v1.Secret)
 	ttl, alreadyExpired := bootstrapsecretutil.GetExpiration(secret, time.Now())
 	if alreadyExpired {
-		klog.V(3).Infof("Deleting expired secret %s/%s", secret.Namespace, secret.Name)
+		klog.V(3).InfoS("Deleting expired Secret", "secretNamespace", secret.Namespace, "secret", secret.Name)
```
Comment (yangjunmyfm192085) [inline pkg/controller/bootstrap/tokencleaner.go:194]: Is klog.KObj or klog.KRef  feasible？

File: pkg/controller/bootstrap/util.go [hunk 4]
```diff
@@ -19,34 +19,34 @@ package bootstrap
...
 	if !ok {
-		klog.V(3).Infof("Invalid secret name: %s. Must be of form %s<secret-id>.", secret.Name, bootstrapapi.BootstrapTokenSecretPrefix)
+		klog.V(3).InfoS("Invalid Secret name. Must be of form <bootstrap-secret-prefix><secret-id>.", "secret", secret.Name, "secretPrefix", bootstrapapi.BootstrapTokenSecretPrefix)
 		return "", "", false
 	}
 
 	tokenID = bootstrapsecretutil.GetData(secret, bootstrapapi.BootstrapTokenIDKey)
 	if len(tokenID) == 0 {
-		klog.V(3).Infof("No %s key in %s/%s Secret", bootstrapapi.BootstrapTokenIDKey, secret.Namespace, secret.Name)
+		klog.V(3).InfoS("No key in Secret", "BootstrapTokenIDKey", bootstrapapi.BootstrapTokenIDKey, "secretNamespace", secret.Namespace, "secret", secret.Name)
```
Comment (yangjunmyfm192085) [inline pkg/controller/bootstrap/util.go:38]: Is klog.KObj or klog.KRef  feasible？

File: pkg/controller/bootstrap/util.go [hunk 5]
```diff
@@ -19,34 +19,34 @@ package bootstrap
...
 	if nameTokenID != tokenID {
-		klog.V(3).Infof("Token ID (%s) doesn't match secret name: %s", tokenID, nameTokenID)
+		klog.V(3).InfoS("Token ID doesn't match Secret name", "tokenID", tokenID, "secret", nameTokenID)
 		return "", "", false
 	}
 
 	tokenSecret = bootstrapsecretutil.GetData(secret, bootstrapapi.BootstrapTokenSecretKey)
 	if len(tokenSecret) == 0 {
-		klog.V(3).Infof("No %s key in %s/%s Secret", bootstrapapi.BootstrapTokenSecretKey, secret.Namespace, secret.Name)
+		klog.V(3).InfoS("No key in Secret", "BootstrapTokenSecretKey", bootstrapapi.BootstrapTokenSecretKey, "secretNamespace", secret.Namespace, "secret", secret.Name)
```
Comment (yangjunmyfm192085) [inline pkg/controller/bootstrap/util.go:49]: Is klog.KObj or klog.KRef  feasible？

File: pkg/controller/bootstrap/bootstrapsigner.go (hunk 1 above)
Comment (scottilee) [reply pkg/controller/bootstrap/bootstrapsigner.go]: Fixed.

File: pkg/controller/bootstrap/tokencleaner.go (hunk 3 above)
Comment (scottilee) [reply pkg/controller/bootstrap/tokencleaner.go]: Fixed.

File: pkg/controller/bootstrap/bootstrapsigner.go (hunk 2 above)
Comment (scottilee) [reply pkg/controller/bootstrap/bootstrapsigner.go]: Fixed.

File: pkg/controller/bootstrap/util.go (hunk 4 above)
Comment (scottilee) [reply pkg/controller/bootstrap/util.go]: Fixed.

File: pkg/controller/bootstrap/util.go (hunk 5 above)
Comment (scottilee) [reply pkg/controller/bootstrap/util.go]: Fixed.

**Comment by scottilee** [comment]
@yangjunmyfm192085 thanks for the review! I've addressed your comments.

**Pull request closed without merging**